gulp/config.js

```diff
@@ -1,11 +1,66 @@
-'use strict';
+var dest = "./dist/assets/";
+var src = "./src/assets";
+var version = "/1.0.0";
 
 module.exports = {
+  concatJS: {
+    documentation: {
+      src: [
+        src + '/js/site/global.js',
+        src + '/js/site/prism.js'
+      ],
+      dest: dest + '/js'
+    },
+    protocol: {
+      src: [
+        src + '/js/protocol/protocol-accordion.js',
+        src + '/js/protocol/protocol-modal.js'
+      ],
+      dest: dest + '/js'
+    }
+  },
+  lintCss: {
+    src: [
+      dest + "/sass/protocol/protocol.css",
+      dest + "/sass/site/site.css"
+    ]
+  },
+  cleanCss: {
+    src: [
+      dest + '/css/protocol.css',
+      dest + '/css/site.css',
+      dest + '/css/prism.css'
+    ],
+    dest: dest + '/css',
+    rename: { suffix: '.min' }
+  },
+  replace: {
+    src: src + '/sass/protocol/protocol.scss',
+    dest: dest + '/css',
+    base: "../../../../",
+    replacement: "../../../bower_components/"
+  },
+  sass: {
+    src: [
+      src + '/sass/protocol/protocol.scss',
+      src + '/sass/site/site.scss',
+      src + '/sass/site/prism.scss'
+    ],
+    dest: dest + '/css/'
+  },
+  uglify: {
```

Comment:
Is uglify something we should handle in this project, or leave to sites that consume Protocol? (I could lean either way here, but perhaps something worth a discussion).